.github/workflows/metrics.yml

```diff
@@ -0,0 +1,32 @@
+name: Scheduled Metrics
+on:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  - cron:  '0 0 * * *'
+
+jobs:
+  metrics:
+    name: Collect and Send Metrics
+    if: github.repository == 'openshift-helm-charts/development'
```

Comment:
I am still not clear on running this workflow on development repo. In the story HELM-193 I see this is the requirement "number of downloads for each chart in our OpenShift Helm Repo".

Reply:
@tisutisu I'm OK on running this on Development repository, since the metrics job is for internal use only and the metrics endpoint is public as well, the release urls are also public, so basically this job can run anywhere we want. @mmulholla I do think we don't need this check if we never promote this job to charts repo, it will only live on development.

Reply:
@dperaza4dustbit The check is designed to prevent it from running in a fork of the repo.

Reply:
got it, lgtm then

Reply:
@dperaza4dustbit @mmulholla thanks for explaining .. I got it... LGTM .. thanks